omod/src/main/webapp/dictionary/concept.jsp

```diff
@@ -227,7 +227,7 @@
                 <c:forEach items="${command.locales}" var="loc">
                     <td class="${loc}">
                         <c:forEach items="${command.conceptAnswersByLocale[loc]}" var="answer">
-                            <a href="concept.htm?conceptId=${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))}"><c:out value="${answer.value}" /> (${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))})</a><br/>
+                            <a href="concept.htm?conceptId=${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))}">${answer.value}(${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))})</a><br/>
```

Comment:
@wluyima The strike off is visible if I remove the <c:out ,This can be seen at https://issues.openmrs.org/secure/attachment/43995/Strikeoff_visible.png